go.mod

```diff
@@ -1,17 +1,23 @@
 module github.com/upfluence/cql
 
-go 1.16
+go 1.21
```

Comment:
1.23 like in the CI ?

Reply:
I don't want to put 1.23, because that'd force all of the projects that use this lib to bump to 1.23, which can be a pain :sweat_smile: